Dockerfile

```diff
@@ -28,11 +29,11 @@ RUN apt update -qq && apt install -qq -y --no-install-recommends \
         build-essential \
         file \
         git \
-        cmake \
```

Comment:
android bundled version is too old, repo like uvwasi use newer version, please revert this part.